### PR TITLE
chore(cookie-policy): reactivate the cookie policy banner (DSP-1727)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DSP-1727

### CODE REVIEW

**Comment by kilchenmann** [comment]
The "history back" button doesn't work anymore. Will fix it...

File: src/app/app.component.html [hunk 1]
```diff
@@ -8,4 +8,16 @@
         <router-outlet></router-outlet>
     </div>
 
+    <!-- cookie information banner -->
+    <div class="cookie-banner" *ngIf="showCookieBanner">
+        <p class="note">
+            This web-application uses cookies to provide you with great user experience. By using the application you
```
Comment (mdelez) [inline src/app/app.component.html:14]: with a greater user experience

**Review by mdelez — CHANGES_REQUESTED** [review]
I think there needs to be a "Reject" button as well, no? There is currently no way to dismiss the banner besides clicking the "Accept" button.

**Comment by kilchenmann** [comment]
> I think there needs to be a "Reject" button as well, no? There is currently no way to dismiss the banner besides clicking the "Accept" button.

But what should happen, when the user click on "reject"? Redirect the user to a google page? Because in this case s/he can't use the app. 
This is anyway a temporary solution only. Daniela and Marchin are working on a better implementation, where the user can select which cookies should be enabled.

File: src/app/app.component.html (hunk 1 above)
Comment (kilchenmann) [reply src/app/app.component.html]: rewritten in fd2ecf2

**Comment by mdelez** [comment]
> > I think there needs to be a "Reject" button as well, no? There is currently no way to dismiss the banner besides clicking the "Accept" button.
> 
> But what should happen, when the user click on "reject"? Redirect the user to a google page? Because in this case s/he can't use the app.
> This is anyway a temporary solution only. Daniela and Marchin are working on a better implementation, where the user can select which cookies should be enabled.

The website should still be useable even if the user rejected all **non-essential** cookies. We can still use cookie that are essential to the website operating. The user just needs to be allowed the reject any cookie that may have identifying data. https://gdpr.eu/cookies/#:~:text=To%20comply%20with%20the%20regulations,cookies%20except%20strictly%20necessary%20cookies.&text=Allow%20users%20to%20access%20your,the%20use%20of%20certain%20cookies

**Comment by kilchenmann** [comment]
> > > I think there needs to be a "Reject" button as well, no? There is currently no way to dismiss the banner besides clicking the "Accept" button.
> > 
> > 
> > But what should happen, when the user click on "reject"? Redirect the user to a google page? Because in this case s/he can't use the app.
> > This is anyway a temporary solution only. Daniela and Marchin are working on a better implementation, where the user can select which cookies should be enabled.
> 
> The website should still be useable even if the user rejected all **non-essential** cookies. We can still use cookie that are essential to the website operating. The user just needs to be allowed the reject any cookie that may have identifying data. https://gdpr.eu/cookies/#:~:text=To%20comply%20with%20the%20regulations,cookies%20except%20strictly%20necessary%20cookies.&text=Allow%20users%20to%20access%20your,the%20use%20of%20certain%20cookies

Yes I know and you're right, but in DSP-APP we have only **essential** cookies (the one for the login). When the user reject them s/he can't use the app. This is why I wrote: "If you want to use the app, you have to accept the cookie policy".

**Comment by mdelez** [comment]
> > > > I think there needs to be a "Reject" button as well, no? There is currently no way to dismiss the banner besides clicking the "Accept" button.
> > > 
> > > 
> > > But what should happen, when the user click on "reject"? Redirect the user to a google page? Because in this case s/he can't use the app.
> > > This is anyway a temporary solution only. Daniela and Marchin are working on a better implementation, where the user can select which cookies should be enabled.
> > 
> > 
> > The website should still be useable even if the user rejected all **non-essential** cookies. We can still use cookie that are essential to the website operating. The user just needs to be allowed the reject any cookie that may have identifying data. https://gdpr.eu/cookies/#:~:text=To%20comply%20with%20the%20regulations,cookies%20except%20strictly%20necessary%20cookies.&text=Allow%20users%20to%20access%20your,the%20use%20of%20certain%20cookies
> 
> Yes I know and you're right, but in DSP-APP we have only **essential** cookies (the one for the login). When the user reject them s/he can't use the app. This is why I wrote: "If you want to use the app, you have to accept the cookie policy".

That's fair. So then at that point I don't think we even need a cookies banner. You could also just add a "Reject non-essential cookies" button that does nothing but dismiss the banner since we don't have any non-essential cookies.

**Comment by kilchenmann** [comment]
> > > > > I think there needs to be a "Reject" button as well, no? There is currently no way to dismiss the banner besides clicking the "Accept" button.
> > > > 
> > > > 
> > > > But what should happen, when the user click on "reject"? Redirect the user to a google page? Because in this case s/he can't use the app.
> > > > This is anyway a temporary solution only. Daniela and Marchin are working on a better implementation, where the user can select which cookies should be enabled.
> > > 
> > > 
> > > The website should still be useable even if the user rejected all **non-essential** cookies. We can still use cookie that are essential to the website operating. The user just needs to be allowed the reject any cookie that may have identifying data. https://gdpr.eu/cookies/#:~:text=To%20comply%20with%20the%20regulations,cookies%20except%20strictly%20necessary%20cookies.&text=Allow%20users%20to%20access%20your,the%20use%20of%20certain%20cookies
> > 
> > 
> > Yes I know and you're right, but in DSP-APP we have only **essential** cookies (the one for the login). When the user reject them s/he can't use the app. This is why I wrote: "If you want to use the app, you have to accept the cookie policy".
> 
> That's fair. So then at that point I don't think we even need a cookies banner. You could also just add a "Reject non-essential cookies" button that does nothing but dismiss the banner since we don't have any non-essential cookies.

As Subotic said yesterday, I should reactivate the banner. So that's what I did.

**Review by mdelez — APPROVED** [review]

**Comment by kilchenmann** [comment]
Thank you @mdelez

**Pull request merged**